5calls/app/src/main/res/xml/settings.xml

```diff
@@ -66,6 +66,23 @@
             app:iconSpaceReserved="false"
             />
 
+        <ListPreference
+            android:key="prefsKeyScriptTextSize"
+            android:title="@string/script_text_size"
+            android:entries="@array/script_text_size_labels"
+            android:entryValues="@array/script_text_size_values"
+            android:defaultValue="16"
```

Comment:
this default 16 is hard-coded in 3 places. Is there any simple way to have a single source of truth? or can this have the default refer to the first item in the entryValues list?